tests/vm/basevm.py

```diff
@@ -122,11 +122,15 @@ def check_sha512sum(fname):
         return fname
 
     def _ssh_do(self, user, cmd, check):
-        ssh_cmd = ["ssh", "-q", "-t",
+        ssh_cmd = ["ssh",
+                   "-t",
                    "-o", "StrictHostKeyChecking=no",
                    "-o", "UserKnownHostsFile=" + os.devnull,
                    "-o", "ConnectTimeout=1",
                    "-p", self.ssh_port, "-i", self._ssh_key_file]
+        # In debug mode, show output of commands.
```

Comment:
I think that comment may be:
# If not in debug mode set ssh quiet mode

Reply:
Good point, will update it.